setup.py

```diff
@@ -85,7 +85,8 @@
             "pytest-xdist>=2.4.0",
         ],
         "train": [
-            "opencv-contrib-python-headless==4.1.2.30",
+            "opencv-python==4.1.2.30",
```

Comment:
is this necessary?

Reply:
Unfortunately, installing `ray[rllib]==1.0.1.post1` appears to end up installing both `opencv-python` and `opencv-python-headless`. The opencv version of `4.1.2.30` was explicitly specified as the later versions seem to be incompatible with `ray[rllib]==1.0.1.post1`.